.gitmodules

```diff
@@ -11,4 +11,5 @@
 	branch = tags/v2.0.0
 [submodule "test_lib/bats-file"]
 	path = test_lib/bats-file
-	url = https://github.com/ztombol/bats-file
+	url = https://github.com/tralston/bats-file
+  branch = tags/v0.3.0
```

Comment:
inconsistent whitespace

Reply:
Great catch - thanks! I've cherry-picked your EditorConfig commit as well into this branch.